src/svg/helper.ts

```diff
@@ -169,4 +169,23 @@ export function getSRTTransformString(
     }
 
     return res.join(' ');
-}
\ No newline at end of file
+}
+
+export const encodeBase64 = (function () {
+    if (typeof window !== 'undefined' && typeof window.btoa === 'function') {
```

Comment:
Can use `util.isFunction(window.btoa)` here

Reply:
I see there is code `typeof window !== 'undefined'` in several places. Should we provide a `hasGlobalWindow` variable to do the check in the `env.ts`?

Reply:
Done. And I'm not sure if it's already enough to assign `hasGlobalWindow` to `true` in [this branch](https://github.com/ecomfe/zrender/pull/856/files#diff-6975367242befd32078190cf2e12e55b9ecc20e377632b2671b260ef3033f263L44).